### PR TITLE
Update Jupyter Data Science Notebooks: Migrated to a community repo

### PR DESCRIPTION
**Pull request by samruddhikhandale**
Ref: https://github.com/devcontainers/devcontainers.github.io/pull/202

As the community template `Jupyter Data Science Notebooks` is now self hosted by the community author, it is no longer needed to be published from this repo.

### CODE REVIEW

**Comment by eitsupi** [comment]
Any update on this?

cc @nathancarter

**Comment by bamurtaugh** [comment]
Hey @minsa110, any concerns with this? I thought you may have also been working on self-publishing a Template for this.

File: .github/workflows/test-pr.yaml [hunk 1]
```diff
@@ -41,7 +41,6 @@ jobs:
             hugo: ./**/hugo/**
             javascript-node-azurite: ./**/javascript-node-azurite/**
             julia: ./**/julia/**
-            jupyter-datascience-notebooks: ./**/jupyter-datascience-notebooks/**
             mit-scheme: ./**/mit-scheme/**
             perl: ./**/perl/**
             puppet: ./**/puppet/**
```
Comment (Maxwell0010) [inline .github/workflows/test-pr.yaml:46]: Pff

**Comment by eitsupi** [comment]
@bamurtaugh @minsa110 Could you take a look at this again? Thanks.

**Comment by minsa110** [comment]
Makes sense to me, thank you for the ping!

**Review by Maxwell0010 — COMMENTED** [review]
y

**Pull request merged**